### ISSUE TITLE
RNA length query

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1975]
still includes these protiens
https://www.pombase.org/results/from/id/0a7c1788-27de-4c0a-87fe-e0536db5e005

**Comment by ValWood** [comment]
needs to exclude mRNA

**Comment by kimrutherford** [comment]
> needs to exclude mRNA

Currently for protein coding genes it queries the length of the translated RNA.  The length of the RNA the makes the product for RNA and coding genes.  Is there any reason to remove that functionality?

That's why the description is:

```
Retrieve genes by spliced RNA length in nucleotides (not including UTRs for protein coding genes):
```

**Comment by kimrutherford** [comment]
> Currently for protein coding genes it queries the length of the translated RNA. 

It also gives us something to put in the column if you add "Spliced RNA length" to a table of protein coding genes:

![image](https://user-images.githubusercontent.com/90474/206415875-46025137-60e9-4481-88e6-80adba7b34e8.png)

**Comment by ValWood** [comment]
Oh I see. But I think a query on the  transcript length of the protein coding portion of a transcript is a bit weird. I think we should restrict this particular  search to ncRNAs ... Whatever we do for proteins will be inconsistent. Sorry I know we discussed this but I didn't think it through... we can discuss next time just to make sure we have the best solution.

@manulera

**Comment by manulera** [comment]
> query on the transcript length of the protein coding portion of a transcript is a bit weird

I agree with this but I guess people can just do a combined query with non-coding genes?

**Comment by ValWood** [comment]
OK! fine as. it is then

**Comment by ValWood** [comment]
closing?
